beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -139,8 +139,9 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 		return validationRes, err
 	}
 
+	var singleAtt *eth.SingleAttestation
```

Comment:
In contrast to the code in `beacon-chain/rpc/prysm/v1alpha1/validator/attester.go`, here we need the variable because `att` becomes the Electra attestation (see line 148). It is necessary to convert before calling `validateUnaggregatedAttWithState`